rewrite-maven/src/main/java/org/openrewrite/maven/cache/RocksdbMavenPomCache.java

```diff
@@ -103,22 +105,23 @@ static synchronized RocksCache getCache(String workspace) {
     CacheResult<MavenMetadata> UNAVAILABLE_METADATA = new CacheResult<>(CacheResult.State.Unavailable, null);
     CacheResult<MavenRepository> UNAVAILABLE_REPOSITORY = new CacheResult<>(CacheResult.State.Unavailable, null);
 
-    public RocksdbMavenPomCache(@Nullable File workspace) {
+    public RocksdbMavenPomCache(@Nullable Path workspace) {
 
         assert workspace != null;
 
-        if(!workspace.exists() && !workspace.mkdirs()) {
-            throw new IllegalStateException("Unable to find or create maven pom cache at " + workspace);
-        } else if (!workspace.isDirectory()) {
-            throw new IllegalStateException("The maven cache workspace must be a directory");
+        File pomCacheDir = new File( workspace.toFile(), ".pom");
```

Comment:
Posterity: The AbstractRewriteMOJO creates a File rather than a Path so it'll need to be updated.
Currently, it either defaults the location to user.home/.rewrite/cache/pom (we'll be removing pom) or uses the provided pomCacheDirectory.

Okay, I'll removed the `.`